examples/full/tools.go

```diff
@@ -6,4 +6,5 @@ package main
 // https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module
 import (
 	_ "github.com/rebuy-de/rebuy-go-sdk/v7/cmd/buildutil"
+	_ "honnef.co/go/tools/cmd/staticcheck"
```

Comment:
I was hoping that adding this to the `tools.go` of the SDK would be enough, but it seems like we have to add it to the `tools.go` of every project. Alternative would be adding it to the Dockerfile and demanding that every dev has the correct version installed.

Reply:
Fine for me this way, I'd rather put it in the tools.go than in the Dockerfile.